backend/uclapi/uclapi/settings.py

```diff
@@ -242,7 +242,7 @@
 #    of a person who has permission to upload new statics to
 #    S3.
 
-if strtobool(os.environ.get("AWS_S3_STATICS", "False")):
+if strtobool(os.environ.get("AWS_S3_STATICS", "False")) or True:
```

Comment:
I assume this line was put in for debug purposes? We don't use AWS S3 statics locally, only in prod / staging.

Reply:
yup yup oops